sysutils/portfile-gen/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        macports macports-contrib pypi2port-20230621
+
+name                portfile-gen
+version             0.2
+revision            0
+categories          sysutils macports
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           darwin
+license             BSD
+supported_archs     noarch
+
+description         Generate a basic template Portfile given a few bits \
+                    of information
+long_description    ${description}.
```

Comment:
Should use the `{*}` operator.

Reply:
Uh...? I thought that was just for replacing the old `glob` syntax?

Reply:
It doesn't replace `glob`, what it does it completely different. It does replace `eval` (as formerly used for compatibility Tcl < 8.5) in some cases like this where you want to append each element of a list to another list rather than appending the entire list as a single element.

Reply:
> It doesn't replace `glob`, what it does it completely different. It does replace `eval` (as formerly used for compatibility with Tcl < 8.5) in some cases like this where you want to append each element of a list to another list rather than appending the entire list as a single element.

So, you mean like in [8491583](https://github.com/macports/macports-ports/pull/20951/commits/84915836d71363d864d0f0cf6a5d0c243e2d2d89)?

Reply:
> > It doesn't replace `glob`, what it does it completely different. It does replace `eval` (as formerly used for compatibility with Tcl < 8.5) in some cases like this where you want to append each element of a list to another list rather than appending the entire list as a single element.
> 
> So, you mean like in [8491583](https://github.com/macports/macports-ports/pull/20951/commits/84915836d71363d864d0f0cf6a5d0c243e2d2d89)?

...assuming so; marking as resolved.